### ISSUE TITLE
Schema facet total does not equal total number of DOIs per repository

### ISSUE THREAD

**Issue by MaryHirsch** [datacite/bracco#443]
### Expected Behaviour

Schema facet shows correct number of DOIs with each schema type

### Current Behaviour

Schema facet total does not equal total number of DOIs per repository

### Steps to Reproduce

See example: https://doi.datacite.org/providers/ala/dois

<img width="339" alt="Screen Shot 2020-07-01 at 15 38 44" src="https://user-images.githubusercontent.com/44582654/86250725-3cb01680-bbb1-11ea-9aec-6652e45704da.png">
<img width="1303" alt="Screen Shot 2020-07-01 at 15 38 51" src="https://user-images.githubusercontent.com/44582654/86250730-3e79da00-bbb1-11ea-94da-1a9531704e4b.png">

### Context (Environment)

Fabrica

### Possible Implementation

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_2of85)

**Comment by kjgarza** [comment]
@MaryHirsch there is a github issue already for this https://github.com/datacite/lupo/issues/485

**Comment by richardhallett** [comment]
@kjgarza Can you provide a summary as to if this issue is still relevant, should this all be handled under https://github.com/datacite/lupo/issues/485 which I note is not marked as a bug?

We basically need one issue for this and a good summary of what the problem is and if known the solution.

**Comment by kjgarza** [comment]
yeah, I think this should be handled in datacite/lupo#485.  Basically, it is needed that all the DOIs are reindexed with the correct schema.
